### PR TITLE
Migrate Azure Private Links article

### LINKED ISSUE #1305: Migrate article: Azure Privatelink
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/5538211

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Resolves #1305. 

Migrating the Azure Private Links instructions; restructured and some minor fixes/rewording.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
@heikju Thank you very much for validating the information in this article. Would you mind reviewing? I just reworded a couple of parts to try to make it clearer when we are referring to Azure vs Aiven functions.

I also have one question about the prerequisites. The original article reads, "Even though the Aiven service nodes have a public IP address, the only "public" access is TCP health probes that[ Azure sends](https://docs.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16) to load balancer target ports from a public IP address. Aiven only allows opening and closing a TCP connection from this address without transmitting any data."

Does this mean that 168.63.129.16 also has to be configured as an allowed IP address in Aiven? 

Thanks!

**Comment by ftisiot** [comment]
Hi @staceysalamon-aiven, could you please review the warnings in https://github.com/aiven/devportal/pull/1451/files and solve them? Most of them are related to capitalization or adding words (like `subnet`) to the dictionary

File: .github/vale/styles/Aiven/capitalization_headings.yml [hunk 1]
```diff
@@ -78,6 +78,7 @@ exceptions:
   - PostgreSQL
   - Postman
   - Premium
+  - Private Link
```
Comment (ftisiot) [inline .github/vale/styles/Aiven/capitalization_headings.yml:81]: should this be `Private Links`

File: docs/platform/howto/use-azure-privatelinks.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
+Use Azure Private Links with Aiven services
+=============================================
+
+Azure Private Link lets you bring your Aiven services into your virtual network over a private endpoint. The endpoint creates a network interface into one of the subnets of the VNet and receives a private IP address from that subnet's IP range. The private endpoint is routed to your Aiven service.
```
Comment (ftisiot) [inline docs/platform/howto/use-azure-privatelinks.rst:4]:
```suggestion
Azure Private Links lets you bring your Aiven services into your virtual network over a private endpoint. The endpoint creates a network interface into one of the subnets of the VNet and receives a private IP address from that subnet IP range. The private endpoint is routed to your Aiven service.
```

File: docs/platform/howto/use-azure-privatelinks.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Prerequisites
+--------------
+
+* Aiven CLI is installed.
```
Comment (ftisiot) [inline docs/platform/howto/use-azure-privatelinks.rst:9]: maybe add a link to the relevant page

File: docs/platform/howto/use-azure-privatelinks.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+* The Aiven service is in a project VPC. This ensures the service is not accessible from the public internet.
```
Comment (ftisiot) [inline docs/platform/howto/use-azure-privatelinks.rst:10]: do we have a document about this?

File: docs/platform/howto/use-azure-privatelinks.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+--------------
+
+* Aiven CLI is installed.
+* The Aiven service is in a project VPC. This ensures the service is not accessible from the public internet. 
+  
+  If you are not using regular VNet peerings, any private IP range can be used for the VPC. There is no network routing between your Azure subscription and the Aiven VPC, so overlapping IP ranges are not an issue.
+
+* The Aiven service is using :doc:`static IP addresses </docs/platform/howto/static-ip-addresses>`.
+
+  Even though services in a VPC only communicate using private IP addresses, Azure load balancers require standard SKU IP addresses for target virtual machines. Azure sends TCP health probes to load balancer target ports from a public IP address.
```
Comment (ftisiot) [inline docs/platform/howto/use-azure-privatelinks.rst:16]: Do we have a link to what is a `SKU IP` ? I dunno :D

**Review by ftisiot — CHANGES_REQUESTED** [review]
Good work Stacey! I left some comments, but this is almost perfect

File: docs/platform/howto/use-azure-privatelinks.rst (hunk 2 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/use-azure-privatelinks.rst:4]: Same as comment above.

File: docs/platform/howto/use-azure-privatelinks.rst (hunk 5 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/use-azure-privatelinks.rst:16]: Good point, I had to Google it. XD 

I will add a link to the relevant Azure article here.

**Comment by ftisiot** [comment]
Hi @staceysalamon-aiven I fixed a couple of links (absolute references should start with `/docs/` instead of `docs/`) and changed some of the sentences to Note or Tips... can you check all the content is still correct?

**Comment by staceysalamon-aiven** [comment]
> Hi @staceysalamon-aiven I fixed a couple of links (absolute references should start with `/docs/` instead of `docs/`) and changed some of the sentences to Note or Tips... can you check all the content is still correct?

Thanks for your patience with this @ftisiot and for all the fixes! The first instruction in Step 2 is about the Azure console not Aiven console, so I've rewritten that to be clearer. 

Let me know what you think! 🙏

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**